hail/src/main/scala/is/hail/expr/ir/functions/RelationalFunctions.scala

```diff
@@ -74,7 +129,8 @@ object RelationalFunctions {
     classOf[MatrixExportEntriesByCol],
     classOf[PCA],
     classOf[VEP],
-    classOf[GetElement]
+    classOf[GetElement],
+    classOf[WrappedMatrixToTableFunction]
```

Comment:
Should WrappedMatrixToValueFunction be here?

Reply:
oh, yes, totally